### PR TITLE
Improvements to the OpenAPI definition re: file content

### PR DESCRIPTION
**Pull request by EricWittmann**
As referenced here:  https://github.com/Apicurio/apicurio-registry/issues/1448

Our OAI description wasn't very good at defining when an operation consumed or produced binary file content.  Should be better now.

### CODE REVIEW

File: app/src/main/resources-unfiltered/META-INF/resources/api-specifications/registry/v2/openapi.json [hunk 1]
```diff
@@ -1139,102 +1154,151 @@
...
+                                                        "property-1": "value1",
+                                                        "property-2": true
+                                                    }
+                                                }
+                                            }
+                                        }
+                                    }
+                                }
+                            }
+                        }
```
Comment (kilahm) [inline app/src/main/resources-unfiltered/META-INF/resources/api-specifications/registry/v2/openapi.json:1275]: Here's another request body that does not define a schema.  This is also the request body I mentioned in #1476

Comment (kilahm) [reply app/src/main/resources-unfiltered/META-INF/resources/api-specifications/registry/v2/openapi.json:1275]: Confirmed this is still causing `openapi-generator` to throw an exception:


```
Exception in thread "main" java.lang.RuntimeException: Could not process operation:
  Tag: class Tag {
    name: Metadata
    description: Sometimes the metadata for an artifact is important. For example, metadata includes 
    when the artifact was created, last updated, and so on. This section contains 
    operations to access (and in some cases change) an artifact's metadata.
    externalDocs: null
}
  Operation: getArtifactVersionMetaDataByContent
  Resource: post /groups/{groupId}/artifacts/{artifactId}/meta
  Schemas: {EditableMetaData=class ObjectSchema {
```

**Comment by kilahm** [comment]
Seems like my jq one liner in #1448  didn't catch all of the operation ids that should define request body schema.

**Review by carlesarnal — APPROVED** [review]
LGTM

**Pull request merged**